subsys/net/lib/nrf_cloud/src/nrf_cloud_transport.c

```diff
@@ -715,14 +618,6 @@ int nct_mqtt_connect(void)
 #else
 		nct.client.transport.type = MQTT_TRANSPORT_NON_SECURE;
 #endif
-#if defined(CONFIG_AWS_FOTA)
-		err = aws_fota_init(&nct.client, aws_fota_cb_handler);
```

Comment:
Is support for FOTA_V1 removed?

Reply:
Can this number be found on the confluence page of FOTA v2? if not add it for reference :)

Reply:
added.
`19 users, averaging 4.5 jobs in 2020`